### PR TITLE
Wiredtiger 2.8.0 (new formula)

### PR DESCRIPTION
**Pull request by jakesjews**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
### Description

An updated version of https://github.com/Homebrew/legacy-homebrew/pull/35039

### CODE REVIEW

File: Formula/wiredtiger.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  desc "high performance NoSQL extensible platform for data management"
+  homepage "http://www.wiredtiger.com"
+  url "https://github.com/wiredtiger/wiredtiger/releases/download/2.8.0/wiredtiger-2.8.0.tar.bz2"
+  sha256 "593e4858a21465db6f8360cc6281489f8a114fa301bd2753c0bde9a86ef107e2"
+
+  depends_on "snappy"
+
+  def install
+    system "./configure", "--with-builtins=snappy,zlib",
+                          "--with-python",
```
Comment (DomT4) [inline Formula/wiredtiger.rb]: Does this require a specific Python version? i.e. at least 2.7, etc?

Comment (jakesjews) [inline Formula/wiredtiger.rb]: I did some digging but couldn't find any required python version in the [WiredTiger docs](http://source.wiredtiger.com/2.8.0/build-posix.html).

Comment (jakesjews) [inline Formula/wiredtiger.rb]: @DomT4 It looks like there isn't a required version of python.

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
Merged as d965249896d32b1b371b95fa18202ff1918d0cc0, thank you!

**Pull request closed without merging**